src/tasks/simulate_uni_auction.ts

```diff
@@ -60,8 +68,9 @@ const simulateETHGNOAuction: () => void = () => {
       "value": "0",
       "data": vesting.interface
         .encodeFunctionData("withdraw", [
-          gnosisDAO.address,
-          hardhatRuntime.ethers.utils.parseEther("285398"),
+          // gnosisDAO.address,
```

Comment:
Ah, I was not aware that we want to push it to: xdaiGovMultisigAddress. You can delete then this comment